salesforce-cdp/1.3/modules/ROOT/pages/salesforce-data-cloud-connector-studio.adoc

```diff
@@ -88,12 +89,32 @@ You can reference a configuration file that contains ANT-style property placehol
 . Click *Test Connection* to confirm that Mule can connect with the specified server.
 . Click *OK*.
 
+[[oauth-client-credentials]]
+=== OAuth Client Credentials
+
+OAuth Client Credentials authentication uses OAuth 2.0 with a token endpoint for authentication.
+
+Enter this information on the *General* tab of the *Global Element Properties* screen to configure OAuth Client Credentials authentication.
+
+[%header,cols="30s,70a"]
+|===
+|Field a|Description
+| Consumer key | Consumer key for the Salesforce-connected app. See the Salesforce documentation for information about generating a consumer key.
+| Consumer secret | Your application's client secret (consumer secret in Remote Access Detail).
+| Token endpoint | URL pointing to the server responsible for providing the authentication token, for example, `\https://domain.my.salesforce.com/services/oauth2/token`.
```

Comment:
Is there an extra backslash at the start of the link?

Reply:
This won't display in the public docs, it's just a formatting thing